src/allomancy/java/leaf/cosmere/allomancy/common/utils/MiscHelper.java

```diff
@@ -47,7 +47,7 @@ public static void consumeNugget(LivingEntity livingEntity, Metals.MetalType met
 					{
 						//todo allomancy godmetal strength
 						final double strength = manifestation.getStrength(iSpiritweb, true);
-						spiritweb.giveManifestation(manifestation, strength < 13 ? 13 : (int) (strength + 1));
+						spiritweb.giveManifestation(manifestation, strength < 16 ? 16 : (int) (strength + 1));
```

Comment:
should add minimum here, if you're okay to add a similar thing to the allomancy config that feruchemy had.